distributed/deploy/local.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import print_function, division, absolute_import
+
+from threading import Thread
+
+from tornado.ioloop import IOLoop
+from tornado.iostream import StreamClosedError
+from tornado import gen
+
+from ..utils import sync, ignoring, All
+from ..executor import Executor
+from ..nanny import Nanny
+from ..scheduler import Scheduler
+from ..worker import Worker, _ncores
+
+
+class Local(object):
+    def __init__(self, n_workers=None, cores_per_worker=None, processes=True, loop=None, start=True, scheduler_port=8786, **kwargs):
```

Comment:
Why did you choose to start processes instead of theads by default? This is not consistent with the default behavior of the `dask-worker` command.